test/approve.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+let co = require("co").wrap
+let assert = require('chai').assert
+let utils = require("./utils/utils.js")
+
+contract("Approving and transferring", function(accounts_) {
+  let TOTAL_SUPPLY
+  let OWNER = accounts_[0]
+  let MIGRATION_MASTER = accounts_[1]
+  let ACCOUNT_ONE = accounts_[2]
```

Comment:
is it just me or are these names pretty confusing? :)
Idea: maybe we can make `MIGRATION_MASTER = accounts_[9]` and then have names match accounts?